matrix-servers

```diff
@@ -0,0 +1,352 @@
+---
```

Comment:
I'm assuming you unintentionally copied this file from `group_vars/matrix-servers` into the base directory.
It can be deleted.

Reply:
deleted. was indeed unintentional.
https://github.com/spantaleev/matrix-docker-ansible-deploy/pull/105/commits/e54f3889663a53f0721e5760d3ee853230e0ac1b